index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
```

Comment:
Please use [inline comments feature](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request) like this one to show the context of your requirements and help them identify where to make the necessary changes 😉

Reply:
Kindly make at least two inline comments in your project files to meet the requirements 📝🤓

Reply:
OPTIONAL
This is just a suggestion so you can apply it if you wish 😁To improve your reviews even further, you could use a couple of emojis and/or gifs in the review. The reason for this is to make the code review more human-friendly 👽🤖

Reply:
Hi @mateo951 ... kindly ask you to help m in discovering how to configure my GitHub in order to have the emoji and icons available as I couldn´t see them.

Reply:
Of course, my pleasure! 🔥😎 Try pressing the Windows logo key + . (period). If you are using other operating systems you could try this [link](https://github.com/ikatyang/emoji-cheat-sheet/blob/master/README.md#flags).